exercises/colorsquare.js

```diff
@@ -54,7 +64,15 @@ class ColorSquare{
 			make sure the rightNeighbor is something!  the rightmost element won't have a neighbor
 		*/
 	handleClick(){
-
+		if (this.colorArray[this.colorIndex + 1]){
```

Comment:
What would happen if the value at this was somehow a falsey value?  like an empty string?